src/WTG.Analyzers.Test/TestData/VisibilityAnalyzer/Partial/Result.cs

```diff
@@ -0,0 +1,23 @@
+namespace ns
+{
+	partial class Foo
+	{
+		private partial int Bar();
+	}
+
+	partial class Foo
+	{
+		private partial int Bar() { return default; }
+	}
+
+	class Outer
+	{
+		partial class Bar
+		{
+		}
+
+		internal partial class Baz
```

Comment:
The internal class would need to be non-nested for it to report a diagnostic. If it's nested, it won't really prove anything.